includes/wp-admin/class-welcome-fields.php

```diff
@@ -0,0 +1,274 @@
+<?php
+/**
+ * ActivityPub Welcome Fields Class.
+ *
+ * @package Activitypub
+ */
+
+namespace Activitypub\WP_Admin;
+
+use Activitypub\Model\Blog;
+use Activitypub\Model\User;
+use Activitypub\Collection\Actors;
+
+use function Activitypub\get_reply_intent_js;
+use function Activitypub\is_user_disabled;
+
+/**
+ * Class Welcome_Fields.
+ */
+class Welcome_Fields {
+	/**
+	 * Initialize the welcome fields.
+	 */
+	public static function init() {
+		add_action( 'load-settings_page_activitypub', array( self::class, 'register_welcome_fields' ) );
+	}
+
+	/**
+	 * Register welcome fields.
+	 */
+	public static function register_welcome_fields() {
+		// Add settings sections.
+		add_settings_section(
```

Comment:
We're a bit inconsistent with backslashing global functions in this file. Not sure if it's worth fixing?